### PR TITLE
Add-readme-to-vuejs-getting-started

### PR DESCRIPTION
**Pull request by berthieresteban**
:warning: depend on #405 

## What does this PR do?
Add readme to vueJS getting started (without vuex)

### How should this be manually tested?
  - Step 1 : clone the documentation repository
  - Step 2 : checkout [377] https://github.com/kuzzleio/documentation/pull/377
  - Step 3 : copy the `/doc/6/getting-started/vuejs` folder of this branch in documentation repo (`/src/sdk/js/6/getting-started`) without:
          - node_modules/
          - package-lock.json
          - /without-vuex/node_modules/
  - Step 4 : add the following line before `dumpToFile: true,` in `/src/.vuepress/config.js` : 
```  exclude: ['/sdk/js/6/getting-started/vuejs/without-vuex/src/**/*'],```
  - Step 5 : (documentation) `npm run dev`
  - Step 6 : [review](http://localhost:8080/sdk/js/6/getting-started/vuejs/without-vuex/)

### Other changes
/
### Boyscout
/

### CODE REVIEW

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+Install the kuzzle-sdk: 
+```bash
+cd kuzzle-playground
+yarn add kuzzle-sdk
+```
+
+In the _App.vue_ file, you should remove the tag, the import and the component registration of the `HelloWorld` component, we won't use it.
+
+## Instancing Kuzzle SDK
```
Comment (Aschen) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
## Instanciating Kuzzle SDK
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+To finish, just add an input binded with the `message` property and a button wich call our `sendMessage()` function:
+
+<<< ./src/App.vue:12[html]
+
+And the following css classe: 
+
+<<< ./src/App.vue:13[css]
+
+You can now add new messages to Kuzzle, and receive the notification of the creation to update your store and display the new messages, enjoy :)
```
Comment (Aschen) [inline doc/6/getting-started/vuejs/without-vuex/index.md]: Maybe you can add a section [Where do we go from here](http://docs.kuzzle.io/sdk/js/6/getting-started/node-js/#where-do-we-go-from-here)
and add a link to the realtime engine guide for example (http://docs.kuzzle.io/core/1/guides/essentials/real-time/

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+To do this, we have to create _src/services/kuzzle.js_ file to put our kuzzle instance, a bit like a singleton:
+
+<<< ./src/services/kuzzle.js
+
+We need to import our Kuzzle SDK instance, so just add the following line in your `<script>` tag in the _App.vue_ file: 
+
+<<< ./src/App.vue:3[js]
+
+Then we will [etablish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exists](sdk/js/6/controllers/index/exists/), create the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) of our tchat.
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
Then we will [etablish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exists](sdk/js/6/controllers/index/exists/), the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) of our chat. 
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+<<< ./src/App.vue:4[js]
+
+Then, create the following functions to fetch and display the messages: 
+
+<<< ./src/App.vue:5[js]
+
+<<< ./src/App.vue:6[js]
+
+The function `fetch_message()` will [search](/sdk/js/6/controllers/document/search/) for hundred newest existing messages and store them in our array before subscribe to notification about `messages` collection. We called it in the `valid()` function we created before.
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
The function `fetch_message()` will [search](/sdk/js/6/controllers/document/search/) for the first hundred newest  messages and store them in our array before subscribe to notification about `messages` collection. We called it in the `valid()` function we created before.
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+To finish this part, just add the following css classes:
+
+<<< ./src/App.vue:9[css]
+
+We can now display the messages stored in Kuzzle. Cool but we need to create some right ?
+
+## Send messages
+
+We must make a simple method that will [create](/sdk/js/6/controllers/document/create/) a new message document in Kuzzle.
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
We need to write a simple method that will [create](/sdk/js/6/controllers/document/create/) a new message document in Kuzzle.
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+<<< ./src/App.vue:10[js]
+
+As you can see we doesn't push the new message in our array on message creation.
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
As you can see we don't push the new message in our array on message creation.
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+Actually, We'll subscribe to the collection that store our messages.
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
Actually, We'll subscribe to the collection that contains our messages.
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+So let's create our `subscribe_messages()` action. It will call the realtime controller of Kuzzle to allow us to [receive notifications](/sdk/js/6/controllers/realtime/subscribe/) on message creation:
+
+<<< ./src/App.vue:11[js]
+
+To finish, just add an input binded with the `message` property and a button wich call our `sendMessage()` function:
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
To finish, just add an input field, bond to the `message` property and a button which calls our `sendMessage()` function:
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+
+<<< ./src/App.vue:12[html]
+
+And the following css classe:
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
And the following CSS class: 
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+
+To finish, just add an input binded with the `message` property and a button wich call our `sendMessage()` function:
+
+<<< ./src/App.vue:12[html]
+
+And the following css classe: 
+
+<<< ./src/App.vue:13[css]
+
+You can now add new messages to Kuzzle, and receive the notification of the creation to update your store and display the new messages, enjoy :)
```
Comment (xbill82) [inline doc/6/getting-started/vuejs/without-vuex/index.md]:
```suggestion
You can now add new messages to Kuzzle and receive the notification of the creation to update your state and display the new messages, enjoy :)
```

File: doc/6/getting-started/vuejs/without-vuex/index.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+Install the kuzzle-sdk: 
+```bash
+cd kuzzle-playground
+yarn add kuzzle-sdk
+```
+
+In the _App.vue_ file, you should remove the tag, the import and the component registration of the `HelloWorld` component, we won't use it.
+
+## Instanciating Kuzzle SDK
```
Comment (thomasarbona) [inline doc/6/getting-started/vuejs/without-vuex/index.md]: better, no?
```suggestion
## Instanciate Kuzzle SDK
```

**Pull request merged**